setup.py

```diff
@@ -62,7 +62,7 @@ def read_version():
       extras_require={
           'test': ['tox', 'flake8', 'pytest==4.4.1', 'pytest-cov', 'pytest-rerunfailures',
                    'pytest-xdist', 'mock', 'tensorflow>=1.3.0', 'contextlib2',
-                   'awslogs', 'pandas']},
+                   'awslogs', 'pandas', 'black==19.3b0 ; python_version >= \'3.6\'']},
```

Comment:
Black is only supported for version 3.6: https://github.com/python/black
Additionally, I specified an exact version because I didn't want a PR build to fail black-check by grabbing a newer black version than was run locally.